hacking/conf2yaml.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+import ast
+import yaml
+import os
+import sys
+from ansible.parsing.yaml.dumper import AnsibleDumper
+
+things = {}
+stuff = {}
+
+op_map = {
+    ast.Add: '+',
+    ast.Sub: '-',
+    ast.Mult: '*',
+    ast.Div: '/',
+}
+
+
+
```

Comment:
That's fine, I didn't realize it was what I wrote until after I got further into the file.